doc/manual/en/ch04_xc_tasks.tex

```diff
@@ -46,9 +49,9 @@ \section{Editing tasks}
 editing prior to flight, and others allow tasks to be modified whilst
 in flight for casual cross-country touring.  Tasks can be saved to
 files and reloaded later, and can be transferred between any XCSoar
-platform (Android, Windows, Linux, ...).
+platform (Android, Windows, Linux\textellipsis).
```

Comment:
This removes the comma. Did you really want to do that?

Reply:
Yes it was. I think a comma is redundant before an ellispsis.
(it will not be the case in math, such as: 1, 2, 3, ... , 4, 5.)